test/landing-page.spec.ts

```diff
@@ -0,0 +1,113 @@
+import * as angular from 'angular';
+import 'angular-mocks';
+import * as jQuery from 'jquery';
+
+import '../src/index';
+import {ComponentTest} from '../test/utils/ComponentTest';
+import {LandingPageController} from '../src/components/landing-page/landing-page.controller';
+
+import 'angular-patternfly';
+import 'angular-animate';
+
+describe('landingPage', () => {
+  var componentTest: ComponentTest<LandingPageController>;
+  var $timeout: any;
+  var $window: any;
+
+  beforeEach( () => {
+    angular.mock.module('webCatalog');
+  });
+
+  beforeEach(() => {
+    inject(function (_$window_: any, _$timeout_: any) {
+      $window = _$window_;
+      $timeout = _$timeout_;
+    });
+  });
+
+  beforeEach(() => {
+    var landingPageHtml: string = '' +
+      '<landing-page search-placeholder="Test Placeholder">' +
```

Comment:
It is building up the html string. I find it easier to understand if it is on multiple lines.